package.json

```diff
@@ -7,9 +7,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/modulesio/archae.git"
+    "url": "https://github.com/toomu/archae.git"
```

Comment:
I dont have permissions to write to https://github.com/modulesio/archae.git . Thought that implied I need to fork it to make the changes